monitor/internal/docker/monitor.go

```diff
@@ -654,14 +657,27 @@ func (d *DockerMonitor) setupDockerDaemon() (err error) {
 func (d *DockerMonitor) waitForDockerDaemon(ctx context.Context) (err error) {
 
 	done := make(chan bool)
-	go func() {
-		for errg := d.setupDockerDaemon(); errg != nil; {
+	go func(gctx context.Context) {
+
+		for {
+			errg := d.setupDockerDaemon()
+			if errg == nil {
+				if err := d.initMonitor(gctx); err != nil {
+					zap.L().Debug("Unable to init monitor")
+				}
+				break
```

Comment:
When we finally succeed, again for user experience, let's put an Info message "Docker monitor initialized"

With these messages, we will know that we are trying .. and whether we succeeded or eventually failed" 
We don't need messages for retry. 

Makes sense?

Reply:
Added a message here as well when we suceed